pageserver/src/tenant/timeline/delete.rs

```diff
@@ -278,6 +281,7 @@ impl DeleteTimelineFlow {
                 // Important. We dont pass ancestor above because it can be missing.
                 // Thus we need to skip the validation here.
                 CreateTimelineCause::Delete,
+                RuntimeAuxFilePolicy::Unspecified, // Aux file policy is not needed for deletion, assuming deletion does not read aux keyspace
```

Comment:
```suggestion
                // Aux file policy is not needed for deletion, assuming deletion does not read aux keyspace
                RuntimeAuxFilePolicy::Unspecified, 
```

Reply:
I am however wondering why add more args to `create_timeline_struct` + `new`, will update this later.

Reply:
The thing I'm unsure about is if we can simply use the `ancestor` + `index_part` to infer the argument, instead of passing it directly.